source/explorer/calendar.html.erb

```diff
@@ -2,10 +2,10 @@
   <ul class="breadcrumbs">
     <li><%= link_to 'Home', '/index.html' %></li>
     <li><%= link_to 'Esploratori', '/esploratori.html' %></li>
-    <li><%= link_to 'Calendario', '/esploratori/calendario.html' %></li>
+    <li><%= link_to 'Calendario Esploratori', '/esploratori/calendario.html' %></li>
```

Comment:
Qua rimuovi "Esploratori" perché mi sono reso conto che in questa pagina ci stavamo già comportando adeguatamente, dato che c'era la voce "Esploratori" poco prima nei breadcrumb che ci permetteva di definire adeguatamente il contesto.